lib/src/document_init_parameters.dart

```diff
@@ -22,74 +22,73 @@ enum NativeImageDecoderSupport {
 
 class DocumentInitParameters {
```

Comment:
`DocumentInitParameters` used by w_viewer to get the `PDFDocumentProxy` object that we use to fetch metadata such as pages, outline, etc. https://github.com/Workiva/W_viewer/blob/055ac07e652e6428d61f7f17205801d56c763d87/lib/src/modules/content_renderer/document_sources/pdfjs_doc_source_base.dart#L149